lib/models/Order.js

```diff
@@ -10,15 +12,40 @@ module.exports = class Order {
   }
 
   static async insert({ product, quantity }) {
-    // TODO: Implement me
+    const { rows } = await pool.query(
+      'INSERT INTO orders(product, quantity) VALUES ($1, $2) RETURNING *;',
+      [product, quantity]
+    );
+    return new Order(rows[0]);
   }
 
   static async getAll() {
-    // TODO: Implement me
+    const { rows } = await pool.query(
+      `
+      SELECT
+      *
+      FROM
+      orders
+      `
+    );
+    return new Order(rows[0]);
```

Comment:
Here you will want to `map` through the rows and return a new `Order` instance for each row.